### ISSUE TITLE
Getting error when trying to run the Automation runbook

### ISSUE THREAD

**Issue by g2901b** [Cj-Scott/Get-AppRegistrationExpiration#15]
Any idea why am I getting the below error? 

_SendToLogAnalytics : System.Net.WebException: The remote server returned an error: (403) Forbidden. at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.GetResponse(WebRequest request) at Microsoft.PowerShell.Commands.WebRequestPSCmdlet.ProcessRecord() At line:138 char:2 + _SendToLogAnalytics -CustomerId $customerId ` + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ + CategoryInfo : NotSpecified: (:) [Write-Error], WriteErrorException + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,_SendToLogAnalytics

**Comment by CJScott88** [comment]
Is the Log analytics workspace Id correct\populated?

**Comment by g2901b** [comment]
Yes, I re-added the Workspace ID as variable in the Automation account variable list and ran the runbook. Got the same error again.  Any thing else you can think of?

**Comment by CJScott88** [comment]
What environment are you running in? the function concatenates the string for commercial URI's but if you are running in Azure Governement for example the URI will need to reflect that.

"https://" + $customerId + ".ods.opinsights.azure.com"

**Comment by g2901b** [comment]
I am running in Azure Public.

**Comment by CJScott88** [comment]
That should be ok then, The only other thing it could be is the shared key. Can you confirm the key was copied appropriately from the Agents Management Primary Key field?
![image](https://user-images.githubusercontent.com/33355109/142433266-95b23563-88f4-4257-8d7f-6836defb8ca6.png)
